### DISCUSSION TITLE
401: Unauthorized

### DISCUSSION THREAD

**Discussion by thomasoggier** [alexkn/node-red-contrib-homeconnect#77] [Q&A]
Hello,
When I try to start the autorization, after get the email and login page, when I select "accept" on the next page, I get "401: Unauthorized".
In scope, I have only put "IdentifyAppliance"

**Comment by alexkn** [comment]
After confirmation you should be redirected to the callback url.
Looks like something is wrong with your callback url or your Node-RED setup.

**Reply by bonstio** [reply]
Did you ever figure this out?

I've hit the same problem. Checked the scope and callback URLs are all good. Deeply puzzled.

**Reply by HeavyHawkZero** [reply]
I have the same issue. Callback URL and credentials are good.
I run the Node Red add-on in Home Assistant with no issues so far.

Strange!
